src/main/resources/AutowiredBeans.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans.xsd
+           http://www.springframework.org/schema/context
+           http://www.springframework.org/schema/context/spring-context.xsd">
+    <context:annotation-config/>
+    <!-- Zdefiniowanie ziarna klawiatura bezargumentowo -->
```

Comment:
Zrobiłbym dwa komentarze: ziarna bez parametrów i z. Następnie po prostu linijki z ziarnami. Może ekstra słowo opisu, ale wydaje mi się, że masz to gdzie indziej.